src/movies.js

```diff
@@ -1,16 +1,89 @@
 // Iteration 1: All directors? - Get the array of all directors.
+
+function getAllDirectors(movies){
+    let answer1 = movies.map((movie) => {
```

Comment:
Try to declare your variables with more meaningful names of what the value inside is. For example, directorsArr